packages/data-provider/src/types/mutations.ts

```diff
@@ -14,23 +15,14 @@ import {
 } from './assistants';
 import { Action, ActionMetadata } from './agents';
 
+// Deprecate custom MutationOptions shape and alias to React Query v5
 export type MutationOptions<
```

Comment:
simplified this type to rely on ts-query. If there's a reason to override it, I can make it more explicit